project_euler/problem_135/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
No need for anything in the `__init__.py` file. This file is supposed to be empty.